subprojects/common/okhttp/src/main/kotlin/com/avito/http/RetryInterceptor.kt

```diff
@@ -42,11 +44,17 @@ class RetryInterceptor constructor(
 
             try {
                 prepareForRetry(response)
+                val request = if (tryCount > 1) {
```

Comment:
Do we modify only the second retry? Why?

Reply:
on second try; we don't modify only original request processing

Reply:
Sorry, missed that we do all `proceed` in the loop.